metadata/metadata-core/src/main/java/com/thinkbiganalytics/metadata/core/feed/InMemoryFeedProvider.java

```diff
@@ -368,6 +371,15 @@ public Feed findById(com.thinkbiganalytics.metadata.api.feed.Feed.ID id) {
         // TODO Auto-generated method stub
         return null;
     }
+    
+    /* (non-Javadoc)
+     * @see com.thinkbiganalytics.metadata.api.BaseProvider#findAll(org.springframework.data.domain.Pageable)
+     */
+    @Override
+    public Page<Feed> findPage(Pageable page, String filter) {
+        // TODO Auto-generated method stub
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)